kura/org.eclipse.kura.deployment.agent/src/main/java/org/eclipse/kura/deployment/agent/impl/DeploymentAgent.java

```diff
@@ -454,13 +462,19 @@ private void addPackageToConfFile(String packageName, String packageUrl) {
 
     private void removePackageFromConfFile(String packageName) {
```

Comment:
The `addPackageToConfFile` and `removePackageFromConfFile` have quite a lot of code in common. Sonar [is whining](https://sonarcloud.io/component_measures?id=org.eclipse.kura%3Akura&metric=new_duplicated_lines_density&branch=PR-4016&view=list) about that. If you have the time and the will to refactor the common code it would be great. If you don't we can leave it as-is.